.github/DISCUSSION_TEMPLATE/rfcs.yml

```
@@ -1,23 +1,27 @@
 title: "[RFC]"
-labels: "[rfc]"
+labels: [rfc]
 body:
-  - type: markdown
-    id: rfc
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: |
+        An abstract, tl;dr or executive summary of your RFC.
```

Nitpick, should this be:
```suggestion
        An abstract, tl;dr, or executive summary of your RFC.
```

---

sorted!